analyses/focal-cn-file-preparation/01-add-ploidy-cnvkit.Rmd

```diff
@@ -17,19 +17,42 @@ library(dplyr)
 ### Read in data
 
 ```{r}
-cnvkit_file <- file.path("..", "..", "data", "cnv-cnvkit.seg.gz")
+cnvkit_file <- file.path("..", "..", "data", "v8", "cnv-cnvkit.seg.gz")
 cnvkit_df <- readr::read_tsv(cnvkit_file)
 ```
 
 ```{r}
-histologies_file <- file.path("..", "..", "data", "histologies.tsv")
+histologies_file <- file.path("..", "..", "data", "v8", "histologies.tsv")
 histologies_df <- readr::read_tsv(histologies_file, guess_max = 100000)
 ```
 
+
+# Edits for https://github.com/PediatricOpenTargets/ticket-tracker/issues/177
+# For PBTA, histologies file has germline_sex_estimate already assigned to WXS patients which have had WGS performed
+# Updating here germline_sex_estimate for cohort=="TARGET" where is.na(germline_sex_estimate)
+#Isolate IDs where germline_sex_estimate is NA and cohort == "TARGET"
+germline_sex_estimate_na <- histologies_df %>% filter(cohort == "TARGET") %>% filter(is.na(germline_sex_estimate))
+na_ids <- germline_sex_estimate_na %>% pull(Kids_First_Biospecimen_ID)
+
+#Split the dataset into two groups, fix data for select IDs
+histology_subset1 <- histologies_df %>% filter(!Kids_First_Biospecimen_ID %in% na_ids)
+
+histology_subset2 <- germline_sex_estimate_na %>% mutate(germline_sex_estimate = reported_gender)
+
+#Merge data after fixing germline_estimate with new value
+
+histologies_df_final <- rbind(histology_subset1,histology_subset2)
```

Comment:
```suggestion
histologies_df_final <- bind_rows(histology_subset1, histology_subset2)
```

rbind is deprecated, so you should switch to `bind_rows` here; also added spacing

Reply:
Implemented the change.